webapp/src/EventListener/AddContentSecurityPolicyListener.php

```diff
@@ -17,7 +17,10 @@ public function __invoke(ResponseEvent $event): void
         // the profiler requires 'unsafe-eval' for script-src 'self'.
         $response = $event->getResponse();
         $cspExtra = $this->profiler ? "'unsafe-eval'" : "";
-        $csp = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' $cspExtra; img-src 'self' data:; worker-src 'self' blob:; font-src 'self' data:;";
+        $csp  = "default-src 'self';";
+        $csp .= "font-src 'self' data:; img-src 'self' data:;";
+        $csp .= "style-src 'self' 'unsafe-inline'; worker-src 'self' blob:";
+        $csp .= "script-src 'self' 'unsafe-inline' $cspExtra;";
```

Comment:
This is an unrelated change and seems to only change formatting. If we do this, lets put each -src on a separate line (but preferably in a separate PR)

Reply:
It's from the merge conflict I got, will fix it with rebasing.